protos/google/api/README.md

```diff
@@ -0,0 +1,46 @@
+## API Protos
+
+This folder contains the schema of the configuration model for Google's
+internal API serving platform, which handles routing, quotas, monitoring,
+logging, and the like.
+
+Google refers to this configuration colloquially as the "service config",
+and the `service.proto` file in this directory is the entry point for
+understanding these.
+
+## Using these protos
+
+To be honest, we probably open sourced way too much of this (basically by
+accident). There are a couple files in here you are most likely to be
+interested in: `http.proto`, `documentation.proto`, `auth.proto`, and
+`annotations.proto`.
```

Comment:
And `field_behavior.proto`, `resource.proto` -- although TypeScript may not actually use either of those.